CinemaAPI/Startup.cs

```diff
@@ -76,12 +75,12 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
-            if (env.IsDevelopment())
-            {
+            //if (env.IsDevelopment())
```

Comment:
Is there a reason this is just commented out? Mark will probably ask us to take it out if it's not necessary.

Reply:
Yes, commenting that out allows swagger to work when it is deployed. Beau wanted to use swagger to test the deployment early on. It is, however, a security vulnerability; so eventually, the commented out code must be un-commented.